packages/lib/server/repository/workflow.ts

```diff
@@ -139,36 +139,64 @@ export class WorkflowRepository {
 
     let verifiedEmails: string[] = [userEmail];
 
+    const secondaryEmails = await prisma.secondaryEmail.findMany({
+      where: {
+        userId,
+        emailVerified: {
+          not: null,
+        },
+      },
+    });
+    verifiedEmails = verifiedEmails.concat(secondaryEmails.map((secondaryEmail) => secondaryEmail.email));
     if (teamId) {
-      const team = await prisma.team.findFirst({
+      const teamMembers = await prisma.user.findMany({
```

Comment:
Since this query is getting nested memberships and secondary emails, refactored to query the user table.